### DISCUSSION TITLE
Other ways to install?

### DISCUSSION THREAD

**Discussion by JoelTech43** [File-New-Project/EarTrumpet#776] [Q&A] [answered]
This is just something that I've wondered, is there a way to install it via a  .exe file?
I know there are 2 other ways but I don't want to get chocolatey and I don't like using the Microsoft store so is there any other way at all?

**Comment by riverar** [comment, marked as answer]
@Coding-Mole Not really. These channels are important and necessary to keep you updated. But if you really insist, you could download the package directly from https://chocolatey.org/packages/eartrumpet/ (bottom right, Download link) or compile it yourself.
